tests/test_table.py

```diff
@@ -341,8 +341,12 @@ def test_update_expressions(TestModel, TestModel_entries, dynamo_local):
 
     two.update(things=['foo'])
     assert two.things == ['foo']
-    two.update(things__append=[1])
-    assert two.things == ['foo', 1]
+    two.update(things__append=['bar'])
+    assert two.things == ['foo', 'bar']
+
+    DT = datetime.datetime(2017, 7, 28, 16, 18, 15, 48, tzinfo=dateutil.tz.tzutc())
+    two.update(created=DT)
+    assert isinstance(two.created, datetime.datetime)
```

Comment:
Have you confirmed that this new test fails without your changes to model.py?

Reply:
Now I have:)

Reply:
```
E       AssertionError: assert False
E        +  where False = isinstance(Decimal('1501287495000000'), <type 'datetime.datetime'>)
E        +    where Decimal('1501287495000000') = <tests.conftest.TestModel object at 0x10d6e2a50>.created
E        +    and   <type 'datetime.datetime'> = datetime.datetime
```